pkg/kubelet/config/file.go

```diff
@@ -54,6 +54,8 @@ func NewSourceFile(path string, period time.Duration, updates chan<- interface{}
 func (s *sourceFile) run() {
 	if err := s.extractFromPath(); err != nil {
 		glog.Errorf("Unable to read config path %q: %v", s.path, err)
+		// Emit an update with an empty PodList to allow FileSource to be marked as seen
+		s.updates <- kubelet.PodUpdate{[]api.BoundPod{}, kubelet.SET, kubelet.FileSource}
```

Comment:
This seems questionable-- perhaps only do this if err is file not found? You don't want to blow away all pods if they put up a non-parsing config.